evm_arithmetization/src/cpu/kernel/asm/hash/sha2/temp_words.asm

```diff
@@ -27,6 +27,6 @@
     // stack: c, a, b, Sigma_0(a)
     %sha2_majority
     // stack: Maj(c, a, b), Sigma_0(a)
-    %add_u32
+    ADD
```

Comment:
There is no need for 32-bit reduction, as the result of this macro is being reduced after (see compression section). Similar reasoning for `%sha2_temp_word1`.